taskboot/build.py

```diff
@@ -148,7 +148,8 @@ def build_compose(target, args):
         # Write the produced image
         if output:
             for tag in tags:
-                docker.save(tag, os.path.join(output, '{}.tar'.format(name)))
```

Comment:
It would be better to concat the tag to the name, so that we have:

- `mozilla/bugbug:latest` :arrow_right: `mozilla/bugbug-latest.tar`
- `mozilla/bugbug:v123` :arrow_right: `mozilla/bugbug-v123.tar`

The artifact would be nicer ! WDYT @marco-c ?

Reply:
Yep, or we could overwrite two images which have the same tag

Reply:
The `tag` variable is actually the full tagged Docker image name, ie `registry.hub.docker.com/mozilla/bugbug-base:latest`. We just need to replace the `/` in the name by something else or writing the file will fails because directory `registry.hub.docker.com` doesn't exists

Reply:
Maybe we could remove the registry from the name? And replace `:` with a safer character (you never know...)

Reply:
I prefer to be explicit, without the registry, img defaulted to `docker.io` which is the old default registry but it conflicted later with the registry check when pushing which checked that images started with the new registry `registry.hub.docker.com`. I will replace `:` with something else

Reply:
I escaped the `:` by `_`. If we want to be 100% safe, we may need to escape more, see https://wincent.com/wiki/Legal_Git_branch_names for example of bad characters that we may wants to escape